config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
+  get 'cards/show'
+
   get 'home', to: 'home#index'
+  get 'cards', to: 'cards#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.